RESTfulAPI.md

```diff
@@ -146,45 +147,47 @@ GET /country/[THREE_LETTER_COUNTRY_CODE]/[national,global,local,blog]
 
 You can put a comma separated list of keyword, and this apply a filter on the returned media.
 
+@vecna: What's the use of this, this filter is an OR filter, so the query above will not return "national blogs" but "national" OR "blogs". I wonder what the purpose of this filtering is?
```

Comment:
is an **AND**, if you specify: "national" comma "blogs" return both. if you don't specify anything, by default, return all the four categories